modules/component-store/src/component-store.ts

```diff
@@ -66,13 +69,17 @@ export class ComponentStore<T extends object> implements OnDestroy {
   readonly state$: Observable<T> = this.select((s) => s);
   private ɵhasProvider = false;
 
+  // Signal of state$
+  private readonly state: Signal<T>;
```

Comment:
should we make the `state` signal public? 👀

```suggestion
  readonly state: Signal<T>;
```

so consumers can use `computed` instead of `selectSignal` if that's a preference:

```ts
@Injectable()
class CounterStore extends ComponentStore<{ count: number }> {
  readonly count = computed(() => this.state().count);
  readonly doubleCount = computed(() => this.count() * 2);
}
```

Reply:
Good catch, yes this should be public